lib/game_stats.rb

```diff
@@ -1,4 +1,35 @@
 
 
 module GameStats
+
+    def count_of_games_by_season 
+        games_per_season = Hash.new(0)
+        @games.each do |game|
+            if games_per_season.has_key?(game.season_id)
+                games_per_season[game.season_id] += 1
```

Comment:
We probably only need line 9 instead of the whole if statement. They way it was set up with Hash.new(0) defaults the new key's value to be 0 and line 9 will increment it by 1 so it will end up being 1.

Reply:
Thanks for pointing that out! I'll make a note to fix that.

Reply:
yeah, good call Chee. I was also talking to Kat yesterday about design principles and she had said it's best to steer clear of nested conditionals and conditionals inside of enumerators whenever possible - good thing to think about moving forward

Reply:
Definitely.